From fdbeb2964e4fcb0d7117dccfa5fa58424df96f2e Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Tue, 1 Sep 2015 13:43:56 +0900
Subject: [PATCH 11/85] arm64: dts: salvatorx: Add serial ports support

Add debug serial-0 (SCIF2) and debug serial-1 (SCIF1) ports
support for salvatorx.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts |   21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts
index d081d3b..e415595 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts
@@ -17,6 +17,7 @@
 
 	aliases {
 		serial0 = &scif2;
+		serial1 = &scif1;
 	};
 
 	chosen {
@@ -35,6 +36,26 @@
 	clock-frequency = <16666666>;
 };
 
+&pfc {
+	scif1_pins: serial1 {
+		renesas,groups = "scif1_data_a", "scif1_clk", "scif1_ctrl";
+		renesas,function = "scif1";
+	};
+
+	scif2_pins: serial0 {
+		renesas,groups = "scif2_data";
+		renesas,function = "scif2";
+	};
+};
+
+&scif1 {
+	pinctrl-0 = <&scif1_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
 &scif2 {
+	pinctrl-0 = <&scif2_pins>;
+	pinctrl-names = "default";
 	status = "okay";
 };
-- 
1.7.9.5

